src/elements/forms/text-field.css

```diff
@@ -3,8 +3,9 @@
   --form-control-corner-radius: var(--component-corner-radius, 2px);
   --form-control-border-width: 1px;
   --form-control-font-size: var(--body-font-size, 1rem);
+  --form-control-line-height: var(--line-height-base, 1.5);
```

Comment:
It seems like this introduces more issues than solves. It makes text-area size calculation more implicit and easy but at the same time, it (maybe) affects the performance of all the inputs adding another variable into calculations.

Reply:
I wouldn't be worried about performance. 🙂